### ISSUE TITLE
Glitch with card's on dark theme

### ISSUE THREAD

**Issue by Imadlekal** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2907]
### Bug explanation

It appears that there is a visual glitch whenever a certain controls are clicked on dark theme (when they're inside a card).

An example:

```
 <Button  Grid.Row="3"  ToolTip="Actualiser"
                                                     Width="24" Height="24"  Command="{Binding RefreshCommand}" Style="{StaticResource MaterialDesignToolForegroundButton}" >
            <materialDesign:PackIcon Visibility="{Binding ElementName=DemoItemsListBox, Path=SelectedItem.AddButton}" HorizontalAlignment="Center" VerticalAlignment="Center" Kind="Refresh"  Width="24" Height="24"></materialDesign:PackIcon>
        </Button>
```

**Results (when inside a card):**

![P2FCXguasE](https://user-images.githubusercontent.com/14929605/194829238-e483150a-ef18-4064-8f2c-d2b75c3bfe2b.gif)

**Results (when I put the exact same code outside of the card):**

![nVxHWSuinA](https://user-images.githubusercontent.com/14929605/194829418-e8956519-5be9-4a17-93fa-3d2f033faf42.gif)

Same thing with a DataGridView when it's inside the card (it's not that apparent, but you can see the glitch on the left side of the sort arrow:

![X332xQt42Y](https://user-images.githubusercontent.com/14929605/194829777-da313dae-e826-4bc7-8bc0-423a48fffcda.gif)

This doesn't happen when the light theme is switched ON. Any thoughts? 

### Version

4.5.0

**Comment by Keboo** [comment]
Interesting, the last time I saw this was with the ripple control inside of a tab header. I suspect it is related to that, but will need to look into it.

**Comment by Xaalek** [comment]
Looks similar to https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2308

**Comment by Imadlekal** [comment]
I'm still not sure what the problem is exactly, but I no longer have the glitch after uninstalling MahApps.

I wasn't using any of their components, except for the MetroWindow.

**Comment by Erapchu** [comment]
Do you have repo with this bug?

**Comment by Nobody84** [comment]
Got the same behaviour.
I'm using MaterialDesignThemes Package  4.6.1

I'm currently very busy, but maybe I'm able to create a repo with the bug this week.

**Comment by Asivaprasadam** [comment]
@Imadlekal, This might be a UI caching issue on WPF.

I had a similar problem and this is how I was able to resolve it. 
Can you try adding this code after `InitializeComponent()` in `MainWindow.xaml.cs` Constructor and see if it resolves the issue!

```cs
/*
 * TLDR: Removes random lines being drawn at grid borders.
 *
 * Ref: https://stackoverflow.com/questions/16177358/wpf-poor-rendering-of-application
 * Try disabling GPU acceleration: RenderOptions.ProcessRenderMode = RenderMode.SoftwareOnly;
 * Sometimes WPF GPU caching does a lot of tricks, and this can help, without degrading much performance IMO.
 */
RenderOptions.ProcessRenderMode = System.Windows.Interop.RenderMode.Default; // <--- Most Relevant
```
